x-pack/legacy/plugins/siem/server/lib/detection_engine/types.ts

```diff
@@ -4,6 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { AlertAction } from '../../../../../../plugins/alerting/common';
+export { AlertAction } from '../../../../../../plugins/alerting/common';
```

Comment:
It looks like we import from both this file and the original type throughout this branch. Should we consolidate to one location?